observability/tracing/serverless-tracing.adoc

```diff
@@ -13,5 +13,5 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
 [id="additional-resources_serverless-tracing"]
 [role="_additional-resources"]
 == Additional resources for {ocp-product-title}
-* link:https://docs.openshift.com/container-platform/latest/distr_tracing/distr_tracing_arch/distr-tracing-architecture.html#distr-tracing-architecture[{DTProductName} architecture]
-* link:https://docs.openshift.com/container-platform/latest/distr_tracing/distr_tracing_install/distr-tracing-installing.html#installing-distributed-tracing[Installing distributed tracing]
+* link:https://docs.openshift.com/container-platform/latest/observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.html[{DTProductName} architecture]
```

Comment:
The links render correctly in the preview. Just curious: Was the document https://docs.openshift.com/container-platform/latest/observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.html checked before by someone else? Or this is the first time? Thanks

Otherwise looks good.

Reply:
Yes! The doc is documented by the OpenShift team and reviewed it. I just added the reference link.